modular/blocksyncer/database/bucket.go

```diff
@@ -40,6 +40,19 @@ func (db *DB) UpdateBucketByNameToSQL(ctx context.Context, bucket *models.Bucket
 	return stat.SQL.String(), stat.Vars
 }
 
+func (db *DB) UpdateBucketOffChainStatus(ctx context.Context, bucket *models.Bucket) (string, []interface{}) {
+	stat := db.Db.Session(&gorm.Session{DryRun: true}).Table((&models.Bucket{}).TableName()).
+		Where("bucket_name = ?", bucket.BucketName).
+		Select("off_chain_status", "update_at", "update_tx_hash", "update_time").
+		Updates(models.Bucket{
```

Comment:
use Update(bucket) would be enough I think?

Reply:
According to the official Gorm documentation, "Updates" is used to update multiple columns, while "Update" is generally used to update a single one.
<img width="934" alt="image" src="https://github.com/bnb-chain/greenfield-storage-provider/assets/122767193/e7b25622-ac8a-4fe2-83dd-48c40b171ad7">

Reply:
fixed